src/operator/numpy/np_init_op.cc

```diff
@@ -47,11 +47,13 @@ inline bool NumpyIndicesShape(const nnvm::NodeAttrs& attrs,
     << "_npi_indices dimensions the number of dim must not be less than 0";
   mxnet::TShape param_dim = param.dimensions;
   if (!shape_is_known(param_dim)) return false;
+  CHECK_LT(param_dim.Size(), INT32_MAX) << "ValueError: np.indices does not support large"
```

Comment:
can you change the message to “does not support input(s) containing elements >=2^31”